src/main/arrpc.ts

```diff
@@ -30,6 +30,9 @@ export async function initArRPC() {
                 .executeJavaScript(`Vesktop.openInviteModal(${JSON.stringify(invite)})`)
                 .then(callback);
         });
+        server.on("link", (data: any, deep_callback: (valid: boolean) => void) => {
+            mainWin.webContents.executeJavaScript(`Vesktop.openDeepLink(${JSON.stringify(data)})`).then(deep_callback);
```

Comment:
i know you copy pasted this from the invite but I will add proper IPC that isn't just eval to main and then you can switch ur PR to that